day01/ex04/ex04.md

```diff
@@ -0,0 +1,99 @@
+# Exercise 04 - Ingest dataset 
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex04              |
+|   Files to turn in :    |                    |
+|   Forbidden function :  |  None              |
+|   Remarks :             |  n/a               |
+
+Now that you know the basics of how Elasticsearch works, you are ready to work with a real dataset !!
+And to make this fun you gonna use the same dataset as for the SQL day so you can see differences between SQL and NoSQL.
+
+There are many ways you can ingest data into Elasticsearch. In the previous exercise, you've seen how to create a document manually.  
+You could do this for every line of the CSV, with a python script for instance that parse the CSV and create a document for each line. There is an Elasticsearch client API for many languages that help to connect to the cluster (to avoid writing http request in python): [Elasticsearch client](https://www.elastic.co/guide/en/elasticsearch/client/index.html).
```

Comment:
```suggestion
You could do this for every line of the CSV, with a python script for instance that parses the CSV and create a document for each line. There is an Elasticsearch client API for many languages that helps to connect to the cluster (to avoid writing HTTP requests in python): [Elasticsearch client](https://www.elastic.co/guide/en/elasticsearch/client/index.html).
```